splice_machine/src/main/java/com/splicemachine/derby/impl/db/SpliceDatabase.java

```diff
@@ -129,7 +128,7 @@ private void setupASTVisitors(LanguageConnectionContext lctx) {
         afterOptVisitors.add(RowLocationColumnVisitor.class);
         afterOptVisitors.add(FixSubqueryColRefs.class);
         afterOptVisitors.add(JoinConditionVisitor.class);
-        afterOptVisitors.add(BatchOnceVisitor.class);
+    //    afterOptVisitors.add(BatchOnceVisitor.class);
```

Comment:
If we remove it from here we won't be using it anymore, right? I'd remove it altogether (BOVisitor, BONode and BOOperation)

Reply:
They are removed.